spec/system/users/edit_spec.rb

```diff
@@ -124,6 +124,30 @@
       expect(page).to have_text("Display name can't be blank")
     end
 
+    it "is not able to update the profile when phone number is too short" do
```

Comment:
some duplication in these tests, might be worth making a `shared_examples` section about it

Reply:
fixed: created a shared_examples module. Really helped with keeping code DRY